misc/atrium-specs/src/jvmMain/kotlin/ch/tutteli/atrium/specs/integration/DateAsLocalDateExpectationsSpec.kt

```diff
@@ -0,0 +1,20 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.toEqual
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.creating.Expect
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDate
+import java.util.Date
+
+abstract class DateAsLocalDateExpectationsSpec(
+    asLocalDate: Expect<Date>.(Expect<LocalDate>.() -> Unit) -> Expect<Date>
+) : Spek({
+    describe("asLocalDate") {
+        it("transformation can be applied and a subsequent assertion made") {
+            val date = Date()
```

Comment:
better use a constant date, this could fail if the test runs at 23:59:59.9xx and now()below then already returns tomorrow

Reply:
Thanks, that's a very good point, I'll change it.